templates/localgov-alert-banner--full.html.twig

```diff
@@ -0,0 +1,55 @@
+{#
+/**
+ * @file localgov_alert_banner.html.twig
+ * Default theme implementation to present Alert banner data.
+ *
+ * This template is used when viewing Alert banner pages.
+ *
+ *
+ * Available variables:
+ * - content: A list of content items. Use 'content' to print all content, or
+ * - attributes: HTML attributes for the container element.
+ * - display_title: Boolean indicating if alert banner title should be displayed
+ * - remove_hide_link: Boolean indicating if the link to close the banner should be hidden
+ * - type_of_alert: Type of alert banner
+ *
+ * @see template_preprocess_localgov_alert_banner()
+ *
+ * @ingroup themeable
+ */
+#}
+{{ attach_library('localgov_alert_banner/alert_banner') }}
+{% set has_link = content.link is not empty %}
+{% set type_of_alert =  type_of_alert|split('--')[1] %}
+{% set classes = [
+    'localgov_alert_banner',
+    'js-alert-banner',
+    type_of_alert ? 'localgov_alert_banner--' ~ type_of_alert : '',
+    is_front ? 'localgov_alert_banner--homepage' : '',
+    has_link ? 'localgov_alert_banner--has-link' : 'localgov_alert_banner--no-link'
+  ]
+%}
+<!-- Alert Banner-->
+<div {{ attributes.addClass(classes) }} role="alert">
+  <div class="wrapper">
+    <div class="localgov_alert_banner--inner">
+      <div class="localgov_alert_banner--inner--content">
+        {% if display_title %}
+          <h2 class="localgov_alert_banner--content-heading">{{ content.title }}</h2>
+        {% endif %}
+        <div class="localgov_alert_banner--content-body">{{ content|without('link', 'title') }}</div>
+        {% if content.link %}
+          <div class="localgov_alert_banner--content-link">{{ content.link }}</div>
+        {% endif %}
+      </div>
+      {% if not remove_hide_link %}
+      <div class="localgov_alert_banner--actions">
+          <div class="localgov_alert_banner--dismiss">
+            <a href="#" id="closeGlobalBanner" class="js-alert-banner-close">Hide <span class="sr-only">Click here to hide this notification</span></a><i class="fas fa-times-circle" aria-hidden="true"></i>
```

Comment:
Please consider making the text [translatable](https://www.drupal.org/docs/theming-drupal/twig-in-drupal/filters-modifying-variables-in-twig-templates#trans):
```
{{ 'Click here to hide this notification' | t }}
```